core/block/import/importer.go

```diff
@@ -131,7 +131,7 @@ func (i *Import) importFromBuiltinConverter(ctx *session.Context,
 	req *pb.RpcObjectImportRequest,
 	c converter.Converter,
 	progress process.Progress) error {
-	allErrors := converter.NewError()
+	allErrors := converter.NewError(0)
```

Comment:
It seems it is better to use explicit value of pb.RpcObjectImportRequest_ALL_OR_NOTHING constant instead of implicit to increase readability. There are 8 places, where NewError is invokated with constant - 3 of them are from tests